### DISCUSSION TITLE
List of all files that have been updated/created in a branch

### DISCUSSION THREAD

**Discussion by cem-al** [jesseduffield/lazygit#2416] [Q&A] [answered]
Is it possible to view all files that have been changed in a branch, not just a single commit?

I am pretty new to Git and would find this useful when reviewing PR's.

**Comment by stefanhaller** [comment, marked as answer]
You can do this by navigating to the last commit before the branch, hit `W` and enter, then navigating back to the tip of the branch (e.g. with `<`). You are now in diff mode, so the main panel shows the entire diff of the whole branch. You can then hit enter to see the list of files changed in the branch.
